### PR TITLE
Add six monts to the default exclude date

### PR DESCRIPTION
**Pull request by DanSallau**
Hi,

Set default date value for self exclusion exclude from feature.

Note: Haven't tested well.

### CODE REVIEW

**Comment by qingweibinary** [comment]
@nuruddeensalihu test failed, npm is quite instable recently

**Comment by nuruddeensalihu** [comment]
@qingweibinary i referenced a yet to exist function. That's why.

File: src/settings/SettingsSelfExclusion.js [hunk 1]
```diff
@@ -34,7 +36,8 @@ export default class SettingsSelfExclusion extends Component {
 			max_30day_losses: props.max_30day_losses,
 			max_open_bets: props.max_open_bets,
 			session_duration_limit: props.session_duration_limit,
-			timeout_until: props.timeout_until,
+			timeout_until_time: props.timeout_until,
```
Comment (qingweibinary) [inline src/settings/SettingsSelfExclusion.js]: why suddenly we are using underscore, instead of camelCase?

Comment (borisyankov) [inline src/settings/SettingsSelfExclusion.js]: This is for API call.

**Comment by borisyankov** [comment]
@nuruddeensalihu Did you use the latest name of the call?

**Pull request merged**